### DISCUSSION TITLE
Fields with relationship shortcut to create the actual resource

### DISCUSSION THREAD

**Discussion by simonecorsi** [payloadcms/payload#782] [Feature Requests & Ideas]
Hey! :wave: First of all nice job!

As the title implies it could be useful, from a UX pov, to have a CTA or something when associating a resource with another collection to quickly "create new" for a quicker workflow!

I meant in field like this:
![Screenshot 2022-07-20 at 16 47 21](https://user-images.githubusercontent.com/5617452/180012336-bbd770c0-64e6-4379-a69b-cb1ba4cab722.jpg)

**Comment by DanRibbens** [comment]
This is on our radar. There are a few great enhancements that need to happen with relationships, this being one of them.

We've got a plan for how this one will work and it is going to be really nice. Stay tuned!

**Comment by jmikrut** [comment]
Hey @simonecorsi — this is now built!

https://github.com/payloadcms/payload/pull/1230

Will be released this week!

**Reply by katywings** [reply]
@jmikrut Fantastic news! 😮👏

**Reply by simonecorsi** [reply]
Great to know thank you! 💪
